docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+# WebRTC
+
+You can have real-time audio/video communication with a remote peer. You can send and receive multimedia sources and generic data with a remote peer. The multimedia sources include audio/video stream from microphone, camera or media file. The generic data includes string or byte data.
+
+The main features of the WebRTC API include:
+
+- Managing media sources
+
+  You can [manage media sources](#media_source) which can be transferred to a remote peer. You can add/remove/mute/pause media sources.
+
+- Controlling data channels
+
+  You can [create/destroy data channels and send/receive generic data](#data_channel) to a remote peer.
+
+- Manipulating state and establishing connection
+
+  You can [use functions for session description and ICE candidates](#establish_connection) to establish network connection with a remote peer properly.
+
+- Rendering audio/video data
+
+  You can [decide how to render audio/video streaming data](#media_render).
+
+## Prerequisites
+
+To use the functions and data types of the WebRTC API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__WEBRTC__MODULE.html), [wearable](../../api/wearable/latest/group__CAPI__MEDIA__WEBRTC__MODULE.html) and [IoT headed](../../api/iot-headed/latest/group__CAPI__MEDIA__WEBRTC__MODULE.html) applications), include the `<webrtc.h>` header file in your application:
+
+```c
+#include <webrtc.h>
+```
+
+<a name="media_source"></a>
+## Managing Media Sources
```

Comment:
For second headings, please use sentence style capitalization
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Managing media sources
```